SeleniumWebDriverBasics/YandexUnitTests.cs

```diff
@@ -0,0 +1,252 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Support.UI;
+using static System.Net.WebRequestMethods;
+
+namespace SeleniumWebDriverBasics
+{
+    [TestFixture]
+    public class YandexUnitTests
+    {
+        private IWebDriver driver;
+        private string baseUrl;
+
+        [SetUp]
+        public void TestSetup()
+        {
+            var service = FirefoxDriverService.CreateDefaultService();
+            this.driver = new FirefoxDriver(service);
+            this.driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            this.baseUrl = "https://mail.yandex.com/";
+            this.driver.Navigate().GoToUrl(this.baseUrl);
+            this.driver.Manage().Window.Maximize();
+        }
+
+        [Test]
+        public void LoginToYandex()
```

Comment:
Use more precise test names. For example, LoginToYandex can be GetLoggedInAccount, GetInfoFromDraftEmail can be GetSubjectAddresseeMessageFromDraftEmail, so on.